overrides/windows-override.json

```diff
@@ -11,6 +11,80 @@
         },
         "elementHiding": {
             "state": "enabled"
+        },
+        "cookie": {
+            "exceptions": [],
+            "state": "enabled",
+            "minSupportedVersion": "0.36.0",
+            "settings": {
+                "trackerCookie": "enabled",
```

Comment:
Is "trackerCookie" and "nonTrackerCookie" a windows only attribute? If so I would suggest slightly renaming it to "trackerCookieBlocking", "nonTrackerCookieBlocking"

Reply:
This (technically) applies to all products, but mostly applies to the browser extensions. Their names should not change at this time.